java/server/src/org/openqa/selenium/grid/commands/sessionmaps.txt

```diff
@@ -0,0 +1,27 @@
+Selenium 4 Grid ships with SessionMap.
+
+A SessionMap is a store of session id and it's capabilities.
+It is a key value store. Now, how that data is stored differs
+on the implementation used. As of Selenium 4, we offer the
+following types of data store.
+
+1. RedisBackedSessionMap
+========================
+
+`RedisBackedSessionMap` uses Redis as datastore for key, value store.
+Below are the steps to start a `RedisBackedSessionMap`.
+
+1. Set an environment variable using `export SESSION_IMPLEMENTATION='org.openqa.selenium.grid.sessionmap.redis.RedisBackedSessionMap'`
```

Comment:
I think there is a typo, should be SESSIONS.

Reply:
Right. Good catch 👍 

Fixed!